contrib/agentsay/README.md

```diff
@@ -0,0 +1,47 @@
+# Cowsay for cf-agents
+
+## Example output
+```
+/-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-\
+|                               |
+|          HELLO, AGENT         |
+|                               |
+|             ● ● ●             |
+|             ● ● ●             |
+|                               |
+|         ●   ● ● ●   ●         |
+|         ●   ● ● ●   ●         |
+|         ●   ● ● ●   ●         |
+|         ●           ●         |
+|             ● ● ●             |
+|             ●   ●             |
+|             ●   ●             |
+|             ●   ●             |
+|                               |
+\-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-/
+```
+
+## Download
+```
+git clone https://github.com/olehermanse/agentsay.git
+cd agentsay
```

Comment:
change Download section to reflect new location?

Reply:
Can't test without merging this first ;) Thanks for pointing it out, I'll create a follow up.